From 448ba0acd5dbe43d1372d190d0e605c7ab31854b Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Sun, 22 Feb 2026 09:36:55 +0800
Subject: [PATCH 2/5] Add support for more handheld devices to fan detection

---
 src/overlay.cpp | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/src/overlay.cpp b/src/overlay.cpp
index fe919a9..f78d0aa 100644
--- a/src/overlay.cpp
+++ b/src/overlay.cpp
@@ -881,7 +881,17 @@ void update_fan(){
       auto dirs = ls(path.c_str(), "hwmon", LS_DIRS);
       for (auto& dir : dirs) {
          string full_path = (path + dir + "/name").c_str();
-         if (read_line(full_path).find("steamdeck_hwmon") != string::npos){
+         string name = read_line(full_path);
+         if (name.find("steamdeck_hwmon") != string::npos ||
+             name.find("asus") != string::npos ||
+             name.find("aynec") != string::npos ||
+             name.find("ayaneo_ec") != string::npos ||
+             name.find("gpdfan") != string::npos ||
+             name.find("msi_ec") != string::npos ||
+             name.find("msi-ec") != string::npos ||
+             name.find("msi_wmi_platform") != string::npos ||
+             name.find("oxp_ec") != string::npos ||
+             name.find("oxpec") != string::npos){
             hwmon_path = path + dir + "/fan1_input";
             break;
          }
-- 
2.53.0

