From 514256ff6d17bf4fbc18856632fb9de8c091317c Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Sat, 20 Sep 2025 22:30:40 +0800
Subject: [PATCH] Add support for gamescope-session-steam

---
 data/user/steamos-manager.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/user/steamos-manager.service b/data/user/steamos-manager.service
index 6a83d2f..8eb9950 100644
--- a/data/user/steamos-manager.service
+++ b/data/user/steamos-manager.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=SteamOS Manager Daemon
-After=gamescope.service
+After=gamescope-session-plus@steam.service
 StartLimitIntervalSec=120
 StartLimitBurst=5
 
-- 
2.51.0

