From 5c300c49d04eff9670c38754c81cfc03b261a722 Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Thu, 9 Apr 2026 20:30:37 +0800
Subject: [PATCH] feat: add repository mirror

---
 src/gamemode-news-hook.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gamemode-news-hook.conf b/src/gamemode-news-hook.conf
index 17f0c16..df65377 100644
--- a/src/gamemode-news-hook.conf
+++ b/src/gamemode-news-hook.conf
@@ -16,4 +16,6 @@ monitor_interval = 5
 # Raw URLs for announcements.json, tried in order (mirror first).
 # announcements.json 的 raw URL，按顺序尝试（镜像优先）。
 [repo_mirrors]
+gitee = https://gitee.com/honjow/gamemode-news/raw/main/announcements.json
+gitea = https://gitea.switchsystem.eu.org/honjow/gamemode-news/raw/branch/main/announcements.json
 github = https://raw.githubusercontent.com/SkorionOS/gamemode-news/master/announcements.json
-- 
2.51.2

