From 0712551b232360631a1245bdba1cf18ade226b1b Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Fri, 29 Aug 2025 03:10:22 +0800
Subject: [PATCH] Add ClawA8 ,new

---
 steamos-power-button.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/steamos-power-button.hwdb b/steamos-power-button.hwdb
index 63fd5b5..7c8ec7a 100644
--- a/steamos-power-button.hwdb
+++ b/steamos-power-button.hwdb
@@ -35,6 +35,7 @@ evdev:name:Power Button:dmi:*svnMicro-StarInternationalCo.,Ltd.:pnClawA1M:*
 evdev:name:Power Button:dmi:*svnMicro-StarInternationalCo.,Ltd.:pnClaw7AI+A2VM:*
 evdev:name:Power Button:dmi:*svnMicro-StarInternationalCo.,Ltd.:pnClaw8AI+A2VM:*
 evdev:name:Power Button:dmi:*svnMicro-StarInternationalCo.,Ltd.:pnClawA8BZ2EM:*
+evdev:name:gpio-keys:dmi:*svnMicro-StarInternationalCo.,Ltd.:pnClawA8BZ2EM:*
 # ONEXPLAYER
 evdev:name:AT Translated Set 2 keyboard:dmi:*svnONE-NETBOOK:*
 evdev:name:Power Button:dmi:*svnONE-NETBOOK:*
-- 
2.51.0

